app.js

```diff
@@ -2,55 +2,26 @@ var fs = require('fs');
 var url = require('url');
 var http = require('http');
 var extend = require('extend');
-var elasticsearch = require('elasticsearch');
 
-var config = require('./config/config.ex.js');
-var localConfig = require('./config/config');
+var cfg = require('./config/cfg');
 
-extend(true, config, localConfig);
+var utils = require('./lib/utils');
 
-// Video ids if elasticsearch is down
-var ids = fs.readFileSync('data').toString().split('\n');
-
-function shuffle(array) {
-    var counter = array.length, temp, index;
-
-    while (counter > 0) {
-        index = Math.floor(Math.random() * counter);
-
-        counter--;
+var es = require('././config/es');
```

Comment:
Yeah, I also thought about this, but forgot to move somewhere :)
I think `data/es` is a better place. Since I wanna implement some kind of bookmarks, it would be appropriate place for data wrappers.